migration/pkg.go

```diff
@@ -2,18 +2,24 @@ package migration
 
 import (
 	"github.com/konveyor/controller/pkg/logging"
-	"github.com/konveyor/tackle2-hub/migration/v1"
 	"github.com/konveyor/tackle2-hub/migration/v2"
 	"github.com/konveyor/tackle2-hub/migration/v3"
+	"github.com/konveyor/tackle2-hub/settings"
 	"gorm.io/gorm"
 )
 
 var log = logging.WithName("migration")
+var Settings = &settings.Settings
 
 //
 // VersionKey is the setting containing the migration version.
 const VersionKey = ".migration.version"
 
+//
+// SupportedFrom is the index of the
+// earliest version that we can migrate from.
+const SupportedFrom = 1
```

Comment:
If we still need the `nil` in the ALL slice, why do we need this ^.  Wondering if the migrator could just skip the nil migrations.  Less to keep in sync.